cognitedata/reveal
# fix: don't visualize annotations before inside 360 image

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-882

## Description :pencil:


Make sure that annotations are hidden until the user is stably situated in the new 360 image.

## How has this been tested? :mag:



- Start Reveal examples
- Open https://localhost:3000/?project=3d-test&env=greenfield-3d&modelId=4923307033982497&revisionId=3100284276970593
- Load 360 images `helideck-site-2-jpeg`
- Enter lower-most 360-image, and the one besides it
- Observe that when moving between 360-images, the annotations do not show before fully inside the new 360 image


## Test instructions :information_source:




## Checklist :ballot_box_with_check:
